account_banking_sepa_credit_transfer/tests/test_sct.py

```diff
@@ -0,0 +1,366 @@
+# Copyright 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# Copyright 2020 Sygel Technology - Valentin Vinagre
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import base64
+import time
+
+from lxml import etree
+
+from odoo.exceptions import UserError
+from odoo.tests.common import SavepointCase
+from odoo.tools import float_compare
+
+
+class TestSCT(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.account_model = cls.env["account.account"]
+        cls.move_model = cls.env["account.move"]
+        cls.journal_model = cls.env["account.journal"]
+        cls.payment_order_model = cls.env["account.payment.order"]
+        cls.payment_line_model = cls.env["account.payment.line"]
+        cls.bank_line_model = cls.env["bank.payment.line"]
+        cls.partner_bank_model = cls.env["res.partner.bank"]
+        cls.attachment_model = cls.env["ir.attachment"]
+        cls.invoice_model = cls.env["account.move"]
+        cls.invoice_line_model = cls.env["account.move.line"]
+        cls.partner_agrolait = cls.env.ref("base.res_partner_2")
+        cls.partner_asus = cls.env.ref("base.res_partner_1")
+        cls.partner_c2c = cls.env.ref("base.res_partner_12")
+        cls.eur_currency = cls.env.ref("base.EUR")
+        cls.usd_currency = cls.env.ref("base.USD")
+        cls.main_company = cls.env["res.company"].create(
+            {"name": "Test EUR company", "currency_id": cls.eur_currency.id}
+        )
+        cls.partner_agrolait.company_id = cls.main_company.id
+        cls.partner_asus.company_id = cls.main_company.id
+        cls.partner_c2c.company_id = cls.main_company.id
+        cls.env.user.write(
+            {
+                "company_ids": [(6, 0, cls.main_company.ids)],
+                "company_id": cls.main_company.id,
+            }
+        )
+        cls.env.ref(
+            "l10n_generic_coa.configurable_chart_template"
```

Comment:
Can we use this without having l10n_generic_coa dependency in the manifest?

Reply:
I think it is correct as it is, if it is added in the manifest the "l10n_generic_coa" will always be installed but it is only used in the test.

Reply:
No, it's not, as this is pre-configuring for the first company that CoA there, provoking problems. Why previous instruction is not working?

Reply:
I don't understand, should I change this?

Reply:
What is the problem with the code in v12?

Reply:
IMO the code is correct, this code fragment is the same in v.12. If it isn't correct, please explain more the error, because I don't understand the previous comments.

Reply:
You're right. It was like that in v12, but it's not the best approach, as the module may be missing and this will fail. Let's keep it this way until further analysis.